edgexfoundry/device-modbus-go
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-modbus-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
1. run edgex non-secure stack
3. build device service- `make build`
4. run device service using config provider
5. verify that private configuration is loaded in consul
6. ensure that writable in the private config is modifyable - change log level and see change in the terminal

## New Dependency Instructions (If applicable)
